src/rgw/rgw_acl_swift.h

```diff
@@ -26,8 +26,10 @@ class RGWAccessControlPolicy_SWIFT : public RGWAccessControlPolicy
   int create(RGWRados *store,
              const rgw_user& id,
              const std::string& name,
+             int& rw_mask,
```

Comment:
COSMETICS: `rw_mask` is an output parameter, and thus it would be nice to have it as the very last one.